### ISSUE TITLE
setup.py has dependencies on other ethereum hf projects

### ISSUE THREAD

**Issue by ericsomdahl** [ethereumproject/pyethereum#5]
Dependencies are hardcoded on 2 other projects:

[Ethash](https://github.com/ethereum/ethash)
[Serpent](https://github.com/ethereum/serpent)

setup.py tries to download built tarballs of these projects -- we need 
1. those projects to be imported into the Classic repo 
2. CI set up

**Comment by ericsomdahl** [comment]
Looks like [Classic Ethash](https://github.com/ethereumproject/ethash) already exists.  No sign there is any CI on the repo though

**Comment by ericsomdahl** [comment]
Serpent is now forked and building the **develop** on TravisCI

[Ethereumproject/Serpent](https://github.com/ethereumproject/serpent/tree/develop)

**Comment by rguiu** [comment]
this can be closed now, but we have a small dependency from a comment to the wiki:

`# see: https://github.com/ethereumproject/pyethapp/wiki/Development:-Versions-and-Releases
`

Should we also form the wiki?

**Comment by whatisgravity** [comment]
If you need servers or domains coordinate with me in the IRC or by email.

**Comment by ericsomdahl** [comment]
@whatisgravity: we did some work while you were out -- hopefully nothing redundant was done.

We recently got bitNovosti to setup build.ethereumclassic.org, pointing at an AWS instance running TeamCity CI. Jetbrains gave us an Unlimited license for TeamCity gratis (along with 16 licenses for their other dev tools.  I have those for anyone who needs something like IDEA, PyCharm, or Webstorm)

EC2 Ubuntu Linux and Windows 2012R2 build agents are set up.  I bought a Mac Mini to act as an OSX agent, which will be co-located once it is set up in a week or so.
